### PR TITLE
Replace hardcoded "-jX" for make

### PR DESCRIPTION
**Pull request by thopiekar**
I've noticed that "make -j5" was set globally when push_arm was done.
There might be other places were -j5 is missing now but I think they can be fixed very quickly.

### CODE REVIEW

**Comment by thopiekar** [comment]
Just added now -j$MAKE_JOBS to all make commands (untested but beleave it works) and replaced all hardcoded -jX options with -j$MAKE_JOBS

File: distribute.sh [hunk 1]
```diff
@@ -93,6 +93,11 @@ case $OSTYPE in
 		;;
 esac
 
+if [ -f /proc/cpuinfo ]; then
```
Comment (kived) [inline distribute.sh]: Maybe this should check if `MAKE_JOBS` is already set, so you can tell it how many jobs to use? Like `if [ -z "$MAKE_JOBS" -a -f /proc/cpuinfo ]; then` ...

Comment (thopiekar) [inline distribute.sh]: Didn't think about this situation. I'm not so good in bash just found a variation of this on the web and rewritten it.
Would it be a problem to merge the PR and you could make this tweak?

**Comment by inclement** [comment]
I guess the only reason this wasn't merged was kived's observation, but it still seems like an improvement so I'll merge it anyway. Sorry for the delay.

**Pull request merged**